assets/figure_ids.nfc

```diff
@@ -503,47 +573,107 @@ Version: 1
 1b92: Greninja
 1bd7: Incineroar
 1d00: Shadow Mewtwo
+1d01: Detective Pikachu
+1d40: Pokemon Trainer
 1f00: Kirby
 1f01: Meta Knight
 1f02: King Dedede
 1f03: Waddle Dee
+1f40: Qbby
 2100: Marth
 2101: Ike
 2102: Lucina
 2103: Robin
 2104: Roy
 2105: Corrin
+2106: Alm
+2107: Celica
 2108: Chrom
+2109: Tiki
 210b: Byleth
 2240: Shulk
+2241: Pyra
 2242: Mythra
 2243: Noah
 2244: Mio
 2280: Ness
 2281: Lucas
-22c0: Chibi Robo
+22c0: Chibi-Robo
 3200: Sonic
 3240: Bayonetta
-3340: Pac-man
+3340: PAC-MAN
+3380: Solaire of Astora
 33c0: Kazuya
-3480: Megaman
+3480: Mega Man
 34c0: Ryu
 34c1: Ken
-3500: Male Hunter
+34c2: Luke
+34c3: Jamie
+34c4: Kimberly
+34c5: Chun-Li
+34c6: Guile
+34c7: Juri
+34c8: Blanka
+34c9: Dhalsim
+34ca: E. Honda
+34cb: Dee Jay
+34cc: Manon
+34cd: Marisa
+34ce: JP
+34cf: Zangief
+34d0: Lily
+34d1: Cammy
+34d2: Rashid
+34d3: A.K.I
+34d4: Ed
+34d5: Akuma
+34d6: M. Bison
+34d8: Elena
+34d9: Sagat
+34da: C. Viper
+34db: Alex
+34dc: Ingrid
+3500: One-Eyed Rathalos
 3501: Nabiru
-3502: Rathian/Cheval
```

Comment:
amiiboapi says the "name" is "Rathian and Cheval", but the "character" is "Rathian".  Do we go by "name", or by "character".  If we go by "character", then "Banjo & Kazooie" is only "Banjo".